hardhat.config.js

```diff
@@ -42,7 +42,15 @@ module.exports = {
         },
         hardhat: {
             allowUnlimitedContractSize: true,
-            accounts
+            accounts,
+            forking: {
+                url: "https://eth-mainnet.g.alchemy.com/v2/" + ALCHEMY_API_KEY,
+                blockNumber: 10647812
```

Comment:
The block number of the original Voting Escrow deployment

Reply:
Needing way to moved this config to config for forking and change it on fly via script.
Good atm.
Ref: Some checks were not successful